api/prisma/schema.prisma

```diff
@@ -133,3 +133,14 @@ model Reporter {
 
   @@map("reporters")
 }
+
+model Error {
+  id        BigInt   @id @default(autoincrement()) @map("error_id")
+  requestId String   @map("request_id")
```

Comment:
I think `requestId` should be defined as `BigInt`. Subgraph collects the data as numbers and when frontend will want to match this request with subgraph request, it would have to convert data type of one of the values.

https://github.com/search?q=repo%3ABisonai%2Forakl-subgraph%20requestId&type=code

Reply:
@martinkersner 

When I tried to make requestId as a `BigInt`, I had a problem because of range of `BigInt` type on `Postgresql`.

As far as I know the range of `BigInt` is 8 bytes long. [Reference](https://www.postgresql.org/docs/14/datatype-numeric.html)
```typescript
bigint | 8 bytes | large-range integer | -9223372036854775808 to +9223372036854775807
```

Since, `requestId` is 77 digits long I had a problem to storing requestId as a `BigInt` and decided to store it as a `string`.

Alternative solution which I found is it to store `requestId` as a `Decimal` type where we can put bignumbers and define how long it can be. 

For example:
```prisma
model Error {
  id        BigInt   @id @default(autoincrement()) @map("error_id")
  requestId Decimal  @map("request_id") @db.Decimal(78, 0)
  timestamp DateTime @db.Timestamptz()
```

What do you think about this?

Reply:
I think the decimal approach is a good one, frontend wont have to convert the datatype.